src/datajudge/utils.py

```diff
@@ -42,3 +44,106 @@ def format_difference(
         f"{s1[:diff_idx]}{_fmt_diff_part(s1, diff_idx)}",
         f"{s2[:diff_idx]}{_fmt_diff_part(s2, diff_idx)}",
     )
+
+
+def util_output_postprocessing_sorter(
```

Comment:
I was goign to suggest a move like this. Now, since these new functions are already in a `utils` namespace, doesn't make much sense to prepend such set of characters for each function.